### PR TITLE
Update site header examples for consistent color usage

### LINKED ISSUE #1223: Update site header examples to use `text-strong`

### PR DESCRIPTION
**Pull request by taysea**
#### What does this PR do?
Updates Header examples throughout the site to have consistent styling. Color of `text-strong` should be used for text. Additionally, some adjustments were made to create consistency between the HPE element and text as well as hit target dimensions for mobile.

[Designs](https://www.figma.com/file/krgk9GTsccqOvcTm8OVzH2/HPE-Header-Component?node-id=0%3A1)

#### Where should the reviewer start?
Any file.

#### What testing has been done on this PR?
Tested locally.

#### What are the relevant issues?
Closes #1223

### CODE REVIEW

File: aries-site/src/examples/cardPreviews/header.js [hunk 1]
```diff
@@ -12,14 +12,15 @@ export const HeaderPreview = () => {
       <Box direction="row">
         <Button icon={<Hpe color="brand" />} tabIndex={-1} />
         <Box gap="xsmall" direction="row">
-          <Text weight="bold" alignSelf="center">
-            {' '}
-            HPE{' '}
+          <Text color="text-strong" weight="bold" alignSelf="center">
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/header.js:15]: Nit. We are repeating the `alignSelf`. Should the box just get `align="center"`?

File: aries-site/src/layouts/main/Header.js [hunk 2]
```diff
@@ -23,11 +22,27 @@ const StyledHeader = ({ ...rest }) => {
...
+            pad={{ vertical: 'small' }}
+            responsive={false}
+          >
+            <Hpe color="brand" />
+            {(size !== 'small' || (size === 'small' && !searchFocused)) && (
+              <Box direction="row" gap="xsmall">
+                <Text color="text-strong" weight="bold">
+                  HPE
+                </Text>
+                <Text color="text-strong">App Name</Text>
```
Comment (halocline) [inline aries-site/src/layouts/main/Header.js:41]: While we're here, can we play with `<Text wordBreak="keep-all" ... />` or a no wrap on the Box? I've never been a fan of when the name does this:
Screen Shot 2020-11-02 at 11 07 38 AM

File: aries-site/src/layouts/main/Header.js [hunk 3]
```diff
@@ -23,11 +22,27 @@ const StyledHeader = ({ ...rest }) => {
       {...rest}
     >
       <Link href="/" passHref>
-        <AppIdentity
```
Comment (halocline) [inline aries-site/src/layouts/main/Header.js:26]: Should we retain an `<AppIdentity />` layout component, even if it is just used for the site? Easy to spot and know what it stands for. Also a pattern we might want to reference when implementing elsewhere and easier to search for AppIdentity.

File: aries-site/src/layouts/content/Example/Example.js [hunk 4]
```diff
@@ -69,6 +70,11 @@ export const Example = ({
   else if (showResponsiveControls) ExampleWrapper = ResponsiveContainer;
   else ExampleWrapper = Box;
 
+  let viewPort;
```
Comment (halocline) [inline aries-site/src/layouts/content/Example/Example.js:73]: 👏

File: aries-site/src/examples/components/header/HeaderActionExample.js [hunk 5]
```diff
@@ -6,11 +6,21 @@ export const HeaderActionExample = () => {
...
+          direction="row"
+          align="center"
+          gap="medium"
+          // pad maintains accessible hit target
+          // non-responsive maintains same dimensions for mobile
+          pad={{ vertical: 'small' }}
+          responsive={false}
+        >
           <Hpe color="brand" />
           <Box direction="row" gap="xsmall">
```
Comment (halocline) [inline aries-site/src/examples/components/header/HeaderActionExample.js:19]: Made a comment regarding wrapping of the App Name, suggesting we investigate whether `<Text wordBreak="keep-all" ... />` or similar might present the App Name better at different viewport widths. If we decide to do that investigation (and it works well), should we also implement in these examples?

File: aries-site/src/examples/components/header/HeaderExample.js [hunk 6]
```diff
@@ -17,6 +17,7 @@ const StyledTextInput = styled(TextInput).attrs(() => ({
 
 export const HeaderExample = () => {
   const size = useContext(ResponsiveContext);
+  console.log({ size });
```
Comment (halocline) [inline aries-site/src/examples/components/header/HeaderExample.js:20]:
```suggestion
```

**Review by halocline — COMMENTED** [review]
Minor comments / suggestions. Otherwise looks good.

File: aries-site/src/examples/cardPreviews/header.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/examples/cardPreviews/header.js:15]: Good catch. Let me check it out.

File: aries-site/src/layouts/main/Header.js (hunk 2 above)
Comment (taysea) [reply aries-site/src/layouts/main/Header.js:41]: Agreed. Let me look into that.

File: aries-site/src/layouts/main/Header.js (hunk 3 above)
Comment (taysea) [reply aries-site/src/layouts/main/Header.js:26]: Yes, that would work.

File: aries-site/src/examples/components/header/HeaderActionExample.js (hunk 5 above)
Comment (taysea) [reply aries-site/src/examples/components/header/HeaderActionExample.js:19]: Yes, we would want to extend across all examples if we find a better implementation.

File: aries-site/src/layouts/main/Header.js (hunk 3 above)
Comment (taysea) [reply aries-site/src/layouts/main/Header.js]: I essentially took what lives in the aries-core file and brought it locally so we're not importing anything from aries-core.

File: aries-site/src/layouts/main/Header.js (hunk 2 above)
Comment (taysea) [reply aries-site/src/layouts/main/Header.js]: The best implementation I could do was wrapping the text to appear as such:
Screen Shot 2020-11-04 at 4 41 54 PM

**Review by halocline — APPROVED** [review]
Good to go.

**Pull request merged**